modepy/quadrature/__init__.py

```diff
@@ -205,6 +205,25 @@ def _quadrature_for_qn(space: QN, shape: Hypercube):
     assert quad.exact_to >= space.order
     return quad
 
+
+@quadrature_for_space.register(TensorProductSpace)
```

Comment:
Done in cb4161b.

Had to modify `quadrature_for_pn` to use Gauss-Legendre quadrature in the 1D case because it was falling back to `XiaoGimbutasSimplexQuadrature`, which is apparently ever so slightly less accurate. Is that ok?

Reply:
That's just fine. I think it might have fallen back to Grundmann-Möller, which is very general, and generally not good. :)